paddle/fluid/imperative/layer.cc

```diff
@@ -53,7 +53,18 @@ static framework::VariableNameMap CreateVarNameMap(
     const framework::OpInfo& op_info, const std::string& op_type,
     const NameVarBaseMap& varbase_map, bool is_input) {
   if (op_info.proto_ == nullptr) {
-    return {};
+    framework::VariableNameMap result;
```

Comment:
Why change this?

Reply:
Grad op will call this function. proto_ of grad op is nulptr